src/components/Attachment/Attachment.js

```diff
@@ -233,46 +252,80 @@ export const renderMedia = (props) => {
  * Attachment - The message attachment
  *
  * @example ../../docs/Attachment.md
- * @type { React.FC<import('types').AttachmentUIComponentProps> }
+ * @type { React.FC<import('types').WrapperAttachmentUIComponentProps> }
  */
 const Attachment = ({
-  attachment,
+  attachments,
   Card = DefaultCard,
   Image = DefaultImage,
+  Gallery = DefaultGallery,
   Audio = DefaultAudio,
   File = DefaultFile,
   Media = DefaultMedia,
   AttachmentActions = DefaultAttachmentActions,
   ...rest
 }) => {
+  const gallery = {
+    type: 'gallery',
+    images: attachments?.filter(
```

Comment:
If attachments can be undefined or null, we have to fix the[ type definition](https://github.com/GetStream/stream-chat-react/pull/513/files#diff-88bec6beae84369c0376b56b3bb88fe1R663) accordingly.